src/models/AssetRewards.ts

```diff
@@ -47,7 +47,9 @@ export default class AssetRewards {
 
     public getAmountsString(): string {
         if (this.rewards.size == 0) return '[]'
-        return '["' + Array.from(this.rewards.values()).join('","') + '"]'
+        const _amounts: string[] = []
+        this.rewards.forEach(value => _amounts.push(value.toFixed()))
+        return '["' + _amounts.join('","') + '"]'
```

Comment:
we can do like this:
```typescript
return this.rewards.length ? JSON.stringify(this.rewards.map(r => r.toString())) : []
```
If what we want is something like this `'["1022","22033","23","40"]'`